samples/MediatR.Examples.PublishStrategies/Publisher.cs

```diff
@@ -78,7 +78,7 @@ private Task ParallelNoWait(IEnumerable<Func<INotification, CancellationToken, T
     {
         foreach (var handler in handlers)
         {
-            Task.Run(() => handler(notification, cancellationToken));
+            var task = Task.Run(() => handler(notification, cancellationToken));
```

Comment:
Why do this?

Reply:
Thankful
Yes, I did because of that
https://github.com/Microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD110.md